app/helpers/investigations/display_text_helper.rb

```diff
@@ -152,6 +152,6 @@ def report_summary_list(investigation)
   end
 
   def has_badges?(investigation)
-    investigation.is_private? || investigation.is_closed? || investigation.coronavirus_related?
+    investigation.is_private? || investigation.is_closed? || investigation.coronavirus_related? || investigation.risk_level&.serious? || investigation.risk_level&.high?
```

Comment:
Suggest using the methods provided by the investigation enum:

```suggestion
    investigation.is_private? || investigation.is_closed? || investigation.coronavirus_related? || investigation.serious? || investigation.high?
```

(It might be worth considering adding [a prefix](https://api.rubyonrails.org/classes/ActiveRecord/Enum.html) to the method names to make this slightly more readable, eg `risk_level_serious?`